time.c

```diff
@@ -3276,8 +3265,8 @@ find_time_t(struct tm *tptr, int utc_p, time_t *tp)
     status = 1;
 
     while (guess_lo + 1 < guess_hi) {
+      binsearch:
         if (status == 0) {
-          binsearch:
```

Comment:
Can't jump here when `status != 0`?

Reply:
Maybe we need to add `status = 0;` just before `goto binsearch;`

Reply:
https://github.com/ruby/ruby/pull/3247/commits/17f0f3b8803d14179dc8b44c79cecf077dee586e